arch/x86/entry/vdso/vclock_gettime.c

```diff
@@ -11,6 +11,8 @@
  * Check with readelf after changing.
  */
 
+#undef CONFIG_KTSAN
```

Comment:
add a comment here
we will have no idea why it is here in a month (and I don't have it now)